backend/Makefile

```diff
@@ -171,7 +171,7 @@ API_LIB_SRC = $(shell find ${ARTEMISAPI} -type f -name '*.py' -o -name '*.cfg')
 ENGINE_BASE := 3.9-slim-bullseye
 
 # Django version
-DJANGO_VER = 3.2.19
+DJANGO_VER = 3.2.20
```

Comment:
Should we bump to the [latest patch](https://docs.djangoproject.com/en/4.2/releases/3.2.23/) of Django 3.2, rather than just the next one?

Reply:
Yea, I think that would be worth doing. Let me get that configured and I will recommit shortly. Thank you for the suggestion!